Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -113,8 +114,10 @@ public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string
 			data.AddPropertyIfValue("settings", settings);
 			data.AddPropertyIfValue("tracking_fields", trackingFields?.Select(tf => new JObject() { { "field", tf.Key }, { "value", tf.Value } }));
 
+			string templateIdUrl = string.IsNullOrWhiteSpace(templateId) ? string.Empty : $"?template_id={templateId}";
+
 			return _client
-				.PostAsync($"users/{userId}/webinars")
+				.PostAsync($"users/{userId}/webinars{templateIdUrl}")
```

Comment:
Here's a simpler way to append querystring parameters:
```suggestion
				.PostAsync($"users/{userId}/webinars")
				.WithArgument("template_id", templateId)
```

Reply:
Great suggestion. I've updated the code. I'm not used to the fluent httpclient you're using.